src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -89,6 +91,71 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 	public previewFeaturesEnabled: boolean = false;
 	public doubleClickEditEnabled: boolean;
 
+	@HostListener('document:keydown', ['$event'])
+	handleKeyboardEvent(event) {
+		const toolbarComponent = (<CellToolbarComponent>this.cellToolbar.first);
+		const toolbarEditCellAction = toolbarComponent.getEditCellAction();
+		let cellEditorProvider = this.cellEditors.find(e => e.cellGuid() === this._model.activeCell.cellGuid);
+		let is_selected = false;
```

Comment:
nit: should be using camel case instead of the underscore here

Reply:
This feedback is outdated but I wanted to mention that I had updated this variable using camelCase.